tests/connect/standalone-x509-auth-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Connect to MongoDB with SSL and X509 auth
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("STANDALONE_X509"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$SSL_DIR = realpath(__DIR__ . '/../../scripts/ssl/');
+
+$driverOptions = [
+    // libmongoc does not allow the hostname to be overridden as "server"
```

Comment:
I thought you were planning on recreating certificates so that we *could* test with the valid hostname checks.